openshift/bc.yml

```diff
@@ -1,24 +1,33 @@
 apiVersion: v1
 kind: Template
+  app: '${PROJECT}-${COMPONENT}'

```

Comment:
This doesn't look right - you'd need a key like `labels` above right?. BUT you can just delete it actually, as the template is only processed locally, no labels needed.